coralproject/talk
# [CORL-439] Allow admins to configure GDPR features

## What does this PR do?
Adds toggles to authentication config panel allowing admins to toggle account features:
- download comments
- change username
- delete account

## Migrations

```js
db.tenants.updateMany({}, {
  $set: {
    accountFeatures: {
      changeUsername: false,
      deleteAccount: false,
      downloadComments: false
    }
  }
})
```

## How do I test this PR?

- log in as admin
- go to configure> authentication
- toggle on and off features
- go to stream
- check for "edit" button next to username and download comments section in settings
